dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -560,7 +562,35 @@ ANNO_TARGETS_JANDEX_DUPLICATE_CLASS.useraction=CMSG0002
 # # {3} A number
 # # {4} The name of a file or the name of a directory
 # # Read - Past tense. 
-ANNO_JANDEX_USAGE=CWWKC0092I: Read Jandex indexes for {0} out of {1} archives ({2} out of {3} classes) in {4}. 
+ANNO_JANDEX_USAGE=CWWKC0092I: Jandex coverage of module {4}: Read Jandex indexes for {0} out of {1} module locations; Jandex indexes provided {2} out of {3} module classes.
 ANNO_JANDEX_USAGE.explanation=This shows the number of archives in the module that were processed by reading Jandex indexes rather than by scanning the actual class files.  It also shows the total number of classes in the application that were processed by reading Jandex indexes.
 ANNO_JANDEX_USAGE.useraction=If Jandex indexes are intended to be read for the application then there is no action.  If Jandex indexes are not to be read for the application, then set the useJandex="false" attribute in the application element in server.xml. Alternatively, set useJandex="false" in the applicationManger element in server.xml to ignore Jandex indexes in all applications.
 
+# # {0} A string: An application name.
+# # {1} A string: A module name.
+# # {2} A number: The total count of module locations.
+# # {3} A number: The count of module locations which had Jandex indexes.
+# # {4} A number: The total count of module classes.
+# # {5} A number: The count of module classes which were read from a Jandex index.
+ANNOCACHE_JANDEX_USAGE=CWWKC0093I: Jandex coverage of module {1} in application {0}: Read Jandex indexes for {3} out of {2} module locations; Jandex indexes provided {5} out of {4} module classes.
+ANNOCACHE_JANDEX_USAGE.explanation=This shows the proportion of a module which was scanned for annotations using Jandex indexes.  Module locations are either the module itself (for example, for an EJB Jar file), or are the WEB-INF/classes folder and WEB-INF/lib Jar files (for a WAR file).
+ANNOCACHE_JANDEX_USAGE.useraction=If Jandex indexes are intended to be read for the application then there is no action.  If Jandex indexes are not to be read for the application, then set the useJandex="false" attribute in the application element in server.xml. Alternatively, set useJandex="false" in the applicationManger element in server.xml to ignore Jandex indexes in all applications.
+
+# # {0} A string: Message text from an exception.
```

Comment:
Change
'{0} A string: Message text from an exception.'
to
'A {0} string: Message text from an exception.'